planner/core/planbuilder.go

```diff
@@ -1833,6 +1833,8 @@ func (b *PlanBuilder) buildDDL(node ast.DDLNode) (Plan, error) {
 	case *ast.RecoverTableStmt:
 		// Recover table command can only be executed by administrator.
 		b.visitInfo = appendVisitInfo(b.visitInfo, mysql.SuperPriv, "", "", "", nil)
+	case *ast.LockTablesStmt:
+		// Nothing need to do.
```

Comment:
need any privelege?

Reply:
According to https://dev.mysql.com/doc/refman/5.7/en/lock-tables.html:

"You must have the `LOCK TABLES` privilege, and the `SELECT` privilege for each object to be locked."

Reply:
Currently we don't have `Lock tables` privilege. How about add this privilege in next PR?

Reply:
Okay